targets/simple_switch/sswitch_CLI.py

```diff
@@ -38,7 +37,7 @@ def handle(*args, **kwargs):
             return f(*args, **kwargs)
         except InvalidMirroringOperation as e:
             error = MirroringOperationErrorCode._VALUES_TO_NAMES[e.code]
-            print "Invalid mirroring operation (%s)" % error
+            print("Invalid mirroring operation (%s)" % error)
```

Comment:
I believe `%` is deprecated now and will be removed in a future version. We can replace it with `format` in a future PR.